src/Sylius/Bundle/CoreBundle/Form/Type/ImageType.php

```diff
@@ -27,7 +28,7 @@
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->addEventSubscriber(new AddCodeFormSubscriber())
+            ->addEventSubscriber(new AddCodeFormSubscriber(TextType::class, ['required' => false]))
```

Comment:
Do we still need this subscriber? It is not mandatory nor unique... Can we just have a `TextType` here? Why should they be not-editable?